packages/commonwealth/client/scripts/views/components/SublayoutHeader/useJoinCommunity.tsx

```diff
@@ -80,6 +82,10 @@ const useJoinCommunity = () => {
     } else {
       setIsAuthModalOpen(true);
     }
+
+    await toggleCommunityStar({
+      community: activeCommunityId,
+    });
```

Comment:
you sure it should be here? It will try to star the community that you did not join yet because either 
- account selector modal will be open (setIsAccountSelectorModalOpen - if you have more than 1 compatible  addresses)
- auth modal will be open (setIsAuthModalOpen - when you try to join without being signed in)